tests/Driver/ChromeDriver/DownloaderTest.php

```diff
@@ -111,6 +158,20 @@ private function mockFsAndArchiveExtractorForSuccessfulDownload(OperatingSystem
         $binaryFilename = 'chromedriver' . ($operatingSystem->equals(OperatingSystem::WINDOWS()) ? '.exe' : '');
 
         $extractPath = sys_get_temp_dir() . DIRECTORY_SEPARATOR . 'chromedriver' . DIRECTORY_SEPARATOR;
+        if ($isJsonVersion) {
+            if ($operatingSystem->equals(OperatingSystem::WINDOWS())) {
+                $extractPath .= 'chromedriver-win32/';
+            }
+
+            if ($operatingSystem->equals(OperatingSystem::MACOS())) {
+                $extractPath .= 'chromedriver-mac-x64/';
+            }
+
+            if ($operatingSystem->equals(OperatingSystem::LINUX())) {
+                $extractPath .= 'chromedriver-linux64/';
+            }
```

Comment:
I feel like this shouldn't be necessary if we rename the dir after downloading, so the rest of the code can function as it did before.